scraper/src/test.rs

```diff
@@ -7,7 +7,7 @@ fn tag_with_newline() {
     let document = Html::parse_fragment(
         r#"
         <a
-                            href="https://github.com/causal-agent/scraper">
```

Comment:
Thanks for your contribution!

I understand that you probably made this via search & replace, but could you please revert the change to the smoke test (the URL is not functional AFAIU) so we can keep it pointing to the original repository as a sort of hat tip to the original author?